.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+*.rst @docs
+blackbox/docs/ @docs
+blackbox/test_docs.py @docs
```

Comment:
Not sure if you need to get notifications for this?

Reply:
I don't need to. but I figure I'm interested in changes to the docs tests setup. this change does not introduce mandatory review or approval

Reply:
Using the CODEOWNER feature to get notification about file changes one is interested in but not responsible for is abusing it IMHO. Please use something like https://app.github-file-watcher.com/ for this.

Reply:
I would argue that the docs team _is_ responsible for the docs tests. it's a joint responsibility with engineering